mobile/lib/common/scaffold_with_nav_bar.dart

```diff
@@ -17,27 +18,30 @@ class ScaffoldWithNavBar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      body: child,
-      appBar: const PreferredSize(
-          preferredSize: Size.fromHeight(40), child: SafeArea(child: AppBarWrapper())),
-      bottomNavigationBar: BottomNavigationBar(
-        items: <BottomNavigationBarItem>[
-          BottomNavigationBarItem(
-            icon: Container(key: tabStable, child: const Icon(Icons.currency_exchange)),
-            label: StableScreen.label,
-          ),
-          BottomNavigationBarItem(
-            icon: Container(key: tabWallet, child: const Icon(Icons.wallet)),
-            label: WalletScreen.label,
-          ),
-          BottomNavigationBarItem(
-            icon: Container(key: tabTrade, child: const Icon(Icons.bar_chart)),
-            label: TradeScreen.label,
-          ),
-        ],
-        currentIndex: _calculateSelectedIndex(context),
-        onTap: (int idx) => _onItemTapped(idx, context),
+    return AnnotatedRegion<SystemUiOverlayStyle>(
+      value: SystemUiOverlayStyle.dark,
```

Comment:
From https://stackoverflow.com/questions/53819118/flutter-where-should-i-call-systemchrome-setsystemuioverlaystylesystemuioverla I understand that we can either set it to dark ... or remove the app bar from all other screens 😅